.github/CODEOWNERS

```diff
@@ -7,6 +7,57 @@
 # Modification to what is considered "core packages" require admin approval.
 /SPECS/core-packages/* @microsoft/cbl-mariner-admins
 
+# Modification to specific packages go to specific teams
+/SPECS/installkernel/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-azure/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-hci/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-headers/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-mshv/* @microsoft/cbl-mariner-kernel
+/SPECS/kernel-uvm/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-hci-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-azure-signed/* @microsoft/cbl-mariner-kernel
+/SPECS-SIGNED/kernel-mstflint-signed/* @microsoft/cbl-mariner-kernel
+
+/SPECS/grub2/* @microsoft/cbl-mariner-bootloader
+/SPECS/grubby/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned-x64/* @microsoft/cbl-mariner-bootloader
+/SPECS/shim-unsigned-aarch64/* @microsoft/cbl-mariner-bootloader
+/SPECS-SIGNED/grub2-efi-binary-signed/* @microsoft/cbl-mariner-bootloader
+
+/SPECS/dracut/* @microsoft/cbl-mariner-dracut
+/SPECS/initramfs/* @microsoft/cbl-mariner-dracut
```

Comment:
Do we want to also add some of the [watched packages](https://github.com/microsoft/CBL-Mariner/blob/2.0/SPECS/initramfs/initramfs.spec#L22) for the initramfs?
```spec
%define watched_pkgs e2fsprogs, systemd, kpartx, device-mapper-multipath, verity-read-only-root, dracut-fips, dracut-megaraid
```

Reply:
Added verity-read-only-root packages